Makefile

```diff
@@ -29,7 +29,7 @@ VERSION_LABEL ?= ${VERSION}
 # The go commands and the minimum Go version that must be used to build the app.
 GO ?= go
 GOFMT ?= $(shell ${GO} env GOROOT)/bin/gofmt
-GO_VERSION_KIALI = 1.17.10
+GO_VERSION_KIALI = 1.18.7
```

Comment:
I see there is now a 1.18.8, but I think we need to use what is available in Brew, right?

Reply:
Yep in brew we have only until 1.18.7